subgraphs/liquity/README.md

```diff
@@ -0,0 +1,9 @@
+# Liquity Subgraph
+
+## Links
+
+- Protocol website: https://www.liquity.org
+- Protocol documentation: https://docs.liquity.org
+- Smart contracts: https://github.com/liquity/dev/tree/main/packages/contracts/contracts
+- Deployed addresses: https://docs.liquity.org/documentation/resources#contract-addresses
+- Existing subgraphs: https://thegraph.com/explorer/subgraph?id=2TmD7Fq3K2BQGmUXNPzrNMXb6PDNbcBWfAHa5o3gRJC8
```

Comment:
Can you include the methodology in this PR: https://github.com/messari/subgraphs/pull/63

You can put it after `Links`

Reply:
Also can you update your smart contract interaction chart to reflect the subgraph implementation and add it here like: https://github.com/messari/subgraphs/blob/master/subgraphs/uniswap-v2/README.md#smart-contracts-interactions